android/app/BUCK

```diff
@@ -0,0 +1,65 @@
+# To learn about Buck see [Docs](https://buckbuild.com/).
```

Comment:
You're using expo, so these /android files should all go to gitignore as far as i know...